lib/stitches/whitelisting_middleware.rb

```diff
@@ -1,29 +1,5 @@
-module Stitches
-  # A middleware that will skip its behavior if the path matches a white-listed URL
-  class WhitelistingMiddleware
-    def initialize(app, options={})
-
-      @app           = app
-      @configuration = options[:configuration] ||  Stitches.configuration
-      @except        = options[:except]        || @configuration.whitelist_regexp
-
-      unless @except.nil? || @except.is_a?(Regexp)
-        raise ":except must be a Regexp"
-      end
-    end
-    def call(env)
-      if @except && @except.match(env["PATH_INFO"])
-        @app.call(env)
-      else
-        do_call(env)
-      end
-    end
+require_relative "allowlist_middleware"
 
-  protected
-
-    def do_call(env)
-      raise 'subclass must implement'
-    end
-
-  end
-end
\ No newline at end of file
+module Stitches
+  WhitelistingMiddleware = AllowlistMiddleware
```

Comment:
This seems pretty straightforward but I haven't seen this pattern before. Do we need to a single happy-path test that this works as expected and there are no unintentional side effects? Though I'm struggling to think of a scenario

Reply:
hmmm.  Yeah, the integration test only runs the generator, but not the code (that would still be using the old middleware).  let me think of a quick test we can do to make sure

Reply:
Actually this might not even be needed.  This class exists only for the two middlewares to inherit from, and they are tested and thus working.

The reason for keeping `WhitelistingMiddleware` alive here is that it's possible someone is using it directly, even though it's not formally part of the public API.

Reply:
If `WhitelistingMiddleware` is _not_ part of the public API, it might be a better long-term decision _not_ to include the alias, so we don't have to support it.